controller.js

```diff
@@ -1,11 +1,11 @@
 window.onload = () => {
-  const navMenu = document.querySelector('.nav-menu');
-  const navItems = document.querySelectorAll('.nav-item');
-  const hamburger = document.querySelector('.nav-toggle');
+  const navMenu = document.querySelector('.nav-menu'); // Get the navigation menu element
+  const navItems = document.querySelectorAll('.nav-item'); // Get all navigation items
+  const hamburger = document.querySelector('.nav-toggle'); // Get the hamburger menu icon
   
   const toggle = e => e.classList.toggle('is-active');
   const toggleNav = ({ target }) => Array.from(navMenu.classList).includes('is-active') ? toggle(navMenu) : null;
 
-  hamburger.addEventListener('click', () => toggle(navMenu, 'is-active'));
```

Comment:
How can I replicate the issue on my end? When I go to https://hassanali.me/studorlio/ using the mobile view, the hamburger menu works for me. I may be missing something.


https://github.com/user-attachments/assets/503d6747-67bc-48ba-ae48-0644fb7ecfef

Reply:
Hello @helfi92 ,


I hope you are doing well.

I've reverted the changes to controller.js and restored it to its previous state. 
And
It looks like the hamburger menu is working fine for me on mobile view, as well and I'm not able to replicate the issue. Since this issue was reported by someone else, it might be specific to certain devices or conditions.

Best Regards

Reply:
Asked the author of the issue for more information - https://github.com/helfi92/studorlio/issues/30#issuecomment-2247920577